synapse/cortex.py

```diff
@@ -30,50 +33,137 @@
 
 logger = logging.getLogger(__name__)
 
-DEFAULT_LAYER_NAME = '000-default'
-
 cmdre = regex.compile(r'^[\w\.]+$')
 
 '''
 A Cortex implements the synapse hypergraph object.
-
-Many Cortex APIs operate on nodes which consist of primitive data structures
-which can be serialized with msgpack/json
-
-Example Node
-
-( (<form>, <valu>), {
-
-    "props": {
-        <name>: <valu>,
-        ...
-    },
-    "tags": {
-
-        "foo": <time>,
-        "foo.bar": <time>,
-    },
-})
 '''
 
-class View:
+class View(s_base.Base):
     '''
     A view represents a cortex as seen from a specific set of layers.
 
     The view class is used to implement Copy-On-Write layers as well as
     interact with a subset of the layers configured in a Cortex.
     '''
-    def __init__(self, core, layers):
+    async def __anit__(self, core, node):
+        await s_base.Base.__anit__(self)
+
         self.core = core
-        self.layers = layers
-        self.model = core.model
 
-        # our "top" layer is "us"
-        self.layer = self.layers[-1]
+        self.node = node
+        self.iden = node.name()
+
+        self.info = await node.dict()
+        self.info.setdefault('owner', 'root')
+        self.info.setdefault('layers', ())
+
+        self.layers = []
+
+        for iden in self.info.get('layers'):
+
+            layr = core.layers.get(iden)
+
+            if layr is None:
+                logger.warning('view %r has missing layer %r' % (self.iden, iden))
```

Comment:
I agree it should die loudly, but we also dont want to prevent cortex startup ( and the availability of the APIs you could use to fix the situation ).  Is this just something that we should change to logger.error() ?